Could not resolve host: github.com

When trying to build checkup with docker as explained in the Readme, i got the following error message (Ubuntu 16.04, Docker 1.11.1):

`
Cloning into '/go/src/github.com/sourcegraph/checkup'...
fatal: unable to access 'https://github.com/sourcegraph/checkup/': Could not resolve host: github.com
package github.com/sourcegraph/checkup/cmd: exit status 128
main.go:3:8: cannot find package "github.com/sourcegraph/checkup/cmd" in any of:
    /usr/local/go/src/github.com/sourcegraph/checkup/cmd (from $GOROOT)
    /go/src/github.com/sourcegraph/checkup/cmd (from $GOPATH)
`

Adding the parameter '--net=host' to the docker call fixed the issue.

--- a/README.md
+++ b/README.md
@@ -313,7 +313,7 @@ Linux binary:
 ```bash
 git clone git@github.com:sourcegraph/checkup.git
 cd checkup
-docker run --rm \ 
+docker run --net=host --rm \ 
 -v `pwd`:/project \ 
 -w /project golang bash \ 
 -c "cd cmd/checkup; go get -v -d; go build -v -ldflags '-s' -o ../../checkup"

